src/db/tables/qa_customs.ts

```diff
@@ -135,12 +135,21 @@ async function addQaCustomBroker(
   }
 }
 
-async function getAllMockedScanResults(): Promise<
-  OnerepScanResultDataBrokerRow[]
-> {
-  const res = (await knex("qa_custom_brokers").select(
-    "*",
-  )) as OnerepScanResultDataBrokerRow[];
+async function getAllMockedScanResults(
+  onerepProfileId: number | null,
+): Promise<OnerepScanResultDataBrokerRow[]> {
+  if (onerepProfileId === null) {
+    logger.error(`onerepProfileId not set`);
+  }
+
+  // Using the onerep_scan_id  as a placeholder for the profile ID
+  const res = (await knex("qa_custom_brokers")
+    .select("*")
+    .where(
+      "onerep_scan_id",
```

Comment:
This is very confusing for maintainability. We should change the name in the table `qa_custom_brokers` from `scan_id` to `profile_id` if that's the place we are storing this:

Specifically using: `table.renameColumn`

We can find examples of similar changes in: `20190328111900_add_email_addresses_table.js`

Reply:
@codemist can add a TODO to the comment with a link to a ticket to fix this? I'm a little confused about the issue, if it's just too much to change right now, or if there's a good idea to use the same value for both IDs. Even in the latter case I think we should spend some more time looking for a better solution here, but that can be its own separate ticket that someone can work on.